assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -413,6 +413,25 @@ che.infra.kubernetes.ingress.annotations_json=NULL
 che.infra.kubernetes.pod.security_context.run_as_user=NULL
 che.infra.kubernetes.pod.security_context.fs_group=NULL
 
+# Number of maximum concurrent async web requests
+# (http requests or ongoing  web socket calls)
+# supported in the underlying shared http client
+# of the `KubernetesClient` instances.
+# Default values are 64, and 5 per-host, which 
+# doesn't seem correct for multi-user scenarios
+# knowing that Che keeps a number of connections
+# opened (e.g. for command or ws-agent logs)
+che.infra.kubernetes.client.http.async_requests.max=1000
+che.infra.kubernetes.client.http.async_requests.max_per_host=1000
+
+# Max number of idle connections in the connection pool
+# of the Kubernetes-client shared http client 
+che.infra.kubernetes.client.http.connection_pool.max_idle=5
+
+# Keep-alive timeout of the connection pool
+# of the Kubernetes-client shared http client
+# in minutes
+che.infra.kubernetes.client.http.connection_pool.keep_alive.mins=5
```

Comment:
We practice separation of measurement unit by underscore instead of a dot

Reply:
Fixed in commit https://github.com/eclipse/che/pull/8892/commits/e88a2649bb2f9387b40739f7bcc52941ec2a8fac